### PR TITLE
add remove object method for Minio

### LINKED ISSUE #4691: minio removeObject method to be added.
Minio works nice @mslourens & @gondar00, thank you for the great work. 

Can you please add the `removeObject` method which can be very similar to getObject method. 

Then we can also delete the objects after processing them.

### PR DESCRIPTION
**Pull request by mslourens**
fixes #4691

### CODE REVIEW

File: plugins/packages/minio/lib/operations.ts [hunk 1]
```diff
@@ -59,3 +59,7 @@ export async function signedUrlForPut(minioClient: MinioClient, queryOptions: ob
 
   return { url };
 }
+
+export async function removeObject(minioClient: MinioClient, queryOptions: object): Promise<object> {
+  return await minioClient.deleteObject(queryOptions['bucket'], queryOptions['objectName']);
```
Comment (gondar00) [inline plugins/packages/minio/lib/operations.ts:64]: Hey @mslourens the method is [`removeObject`](https://github.com/minio/minio-js/blob/master/examples/remove-object.js)
```suggestion
  return await minioClient.removeObject(queryOptions['bucket'], queryOptions['objectName']);
```

**Review by gondar00 — CHANGES_REQUESTED** [review]
Need to add another case to the switch statement for `remove_object` [here](https://github.com/ToolJet/ToolJet/blob/develop/plugins/packages/minio/lib/index.ts#L14)

File: plugins/packages/minio/lib/operations.ts (hunk 1 above)
Comment (mslourens) [reply plugins/packages/minio/lib/operations.ts]: Thanks for the review! I implemented your comments.

**Comment by gondar00** [comment]
Thank you for contributing to Tooljet @mslourens 💯

**Review by gondar00 — APPROVED** [review]

**Pull request merged**